librepo/downloader.c

```diff
@@ -1295,7 +1295,7 @@ static gboolean
 check_zck(LrTarget *target, GError **err)
 {
     assert(!err || *err == NULL);
-    assert(target && target->f && target->target);
+    assert(target && target->fd && target->target);
```

Comment:
Again. fd == 0 is a valid value. Compare to -1.

Reply:
fixed